ee/LICENSE

```diff
@@ -1,5 +1,5 @@
 The PostHog Enterprise license (the “Enterprise License”)
-Copyright (c) 2020-2023 PostHog Inc.
+Copyright (c) 2020-2025 PostHog Inc.
```

Comment:
Well done sir, buy yourself a limited edition data funnel: https://posthog.com/merch

Reply:
data funnel is out of stock 😢 , I demand the signed founders photo instead 😂